worlds/alttp/Rules.py

```diff
@@ -38,37 +45,31 @@ def set_rules(world):
     global_rules(world, player)
     dungeon_boss_rules(world, player)
 
-    if world.mode[player] != 'inverted':
+    if not world.world_state[player].inverted:
         default_rules(world, player)
 
-    if world.mode[player] == 'open':
+    if world.world_state[player] == WorldState.option_open:
         open_rules(world, player)
-    elif world.mode[player] == 'standard':
+    elif world.world_state[player] == WorldState.option_standard:
         standard_rules(world, player)
-    elif world.mode[player] == 'inverted':
+    elif world.world_state[player].inverted:
         open_rules(world, player)
         inverted_rules(world, player)
     else:
-        raise NotImplementedError(f'World state {world.mode[player]} is not implemented yet')
+        raise NotImplementedError(f'World state {world.world_state[player]} is not implemented yet')
```

Comment:
This exception doesn't need to exist since it's not even possible to get here without one of those three, so it should just be removed.